doc/user-docs/ThirdPartyToolProfiles.md

```diff
@@ -51,6 +51,17 @@ Note that the starting directory of Cygwin is set as it is to make the path
 work. The default directory opened when starting Cygwin will be `$HOME` because
 of the `--login` flag.
 
+## Far Manager
+
+```json
+{
+    "name" : "Far",
+    "commandline" : "cmd.exe /c \"c:\\program files\\far manager\\far.exe\"",
```

Comment:
Can't just launch straight into far.exe?

Reply:
Yeah, wut

Reply:
Actually, yes it can.